src/task_scl_structural_habitat.py

```diff
@@ -23,6 +23,11 @@ class str_hab(SCLTask):
             "ee_path": "projects/SCL/v1/Panthera_tigris/zones",
             "static": True,
         },
+        "lc_elev_reclass": {
+            "ee_type": SCLTask.FEATURECOLLECTION,
+            "ee_path": "projects/SCL/v1/Panthera_tigris/source/landcover_reclass_lookup/lc_reclass_test",  # need final table
```

Comment:
I know the data are incomplete now, but we should name this without `_test`, and remove the comment. The code will work with whatever data are in there.

Reply:
Will do.